docs/user/tutorials/01-10-external-authorization-provider.md

```diff
@@ -0,0 +1,178 @@
+# External authorization provider
+
+This tutorial shows how to expose and secure an HTTPBin service using an external authorization provider and the Kyma domain.
+
+## Prerequisites
+
+* Kyma installation with the Istio module enabled.
+* Manually install oauth2-proxy, for example, [using Helm](https://github.com/oauth2-proxy/manifests/tree/main/helm/oauth2-proxy).
```

Comment:
@werdes72 do we need this a prerequisite since we are installing it with custom values in `Install and configure oauth2-proxy` point 3 - https://github.com/kyma-project/istio/blob/1e633c5b37d2aef0773ca4afa29fed95e252aefd/docs/user/tutorials/01-10-external-authorization-provider.md#install-and-configure-oauth2-proxy

Reply:
perhaps just adding the repo:
```bash
helm repo add oauth2-proxy https://oauth2-proxy.github.io/manifests
```

Reply:
Moved and rephrased